### ISSUE TITLE
Cannot read property 'Descendant' of undefined when update to webpack 5.71.0

### ISSUE THREAD

**Issue by jquintozamora** [webpack/webpack#15625]
When updating to webpack 5.71.0, then I'm having this issue:

[webpack-cli] TypeError: Cannot read property 'Descendant' of undefined
at Object. (/home/vsts/work/1/s/node_modules/css-select/lib/compile.js:36:56)
at Module._compile (internal/modules/cjs/loader.js:1085:14)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
at Module.load (internal/modules/cjs/loader.js:950:32)
at Function.Module._load (internal/modules/cjs/loader.js:790:12)
at Module.require (internal/modules/cjs/loader.js:974:19)
at require (internal/modules/cjs/helpers.js:101:18)
at Object. (/home/vsts/work/1/s/node_modules/css-select/lib/index.js:29:17)
at Module._compile (internal/modules/cjs/loader.js:1085:14)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)

With webpack 5.70.0 it works fine

**Comment by jquintozamora** [comment]
Sorry for confusion, solved, it was related to a different update

**Comment by HudsonGraeme** [comment]
> Sorry for confusion, solved, it was related to a different update

Just curious, what ended up causing the error? I am running into this same error and haven't found any other information on it

**Comment by alexander-akait** [comment]
As you can see problem somewhere in `css-select`, check your packages on using `css-select` and you can find a bug

**Comment by yusaira-khan** [comment]
FYI, removing `css-minimizer-webpack-plugin` worked for me.

**Comment by alexander-akait** [comment]
@yusaira-khan But you will have unminified CSS

**Comment by cetorres** [comment]
I'm using this module on my app: `html-webpack-plugin` that has this dependency:
html-webpack-plugin@5.5.0
  └─┬ pretty-error@4.0.0
    └─┬ renderkid@3.0.0
      └── css-select@4.3.0

I believe `renderkid` is still using an older version of `css-select` 4.3.0 that is not compatible with Node 16. There is a newer version of [css-select](https://www.npmjs.com/package/css-select) 5.1.0 that is now updated and in TS that probably works with Node 16.
I created an [issue on renderkid Github repo](https://github.com/AriaMinaei/RenderKid/issues/29) asking the author to try to fix it.
